src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -1,4 +1,4 @@
-from ..util import TestConfig
+from ..util import ConfigTests
```

Comment:
The remaining changes go from `TestConfig` -> `ConfigTests` in the rest of these files.